src/extensions/default/CodeFolding/main.less

```diff
@@ -63,11 +57,11 @@
     border: 1px solid @color-marker-border;
     color: @color-marker-text;
     background-color: @color-marker-bg;
-    
+
     &:after {
         content: "...";
         font-size: 1em;
         vertical-align: top;
         line-height: 0.8em;
     }
-}
```

Comment:
and I actually rm'ed the main.css file, did you mean not to modify main.css at all (by remove from PR)?